tests/transforms/test_qcut.py

```diff
@@ -2468,6 +2468,118 @@ def cut_circuit(x):
         assert res.shape == (shots, 2)
         assert type(res) == np.ndarray
 
+    @pytest.mark.parametrize(
+        "interface_import,interface",
+        [
+            ("autograd.numpy", "autograd"),
```

Comment:
autograd fails for some reason :thinking:

Reply:
`assert <class 'pennylane.numpy.tensor.tensor'> == <class 'numpy.ndarray'>` the result is a pennylane tensor (left) and the input is an nd array, would have thought this is being converted using `qml.math.convert_like`.

Reply:
For `assert type(res) == type(convert_input)`, I believe `convert_input` is an instance of autograd's version of `np.ndarray`. Then `res` is an instance of `pennylane.numpy.tensor.tensor` which is a subclass of autograd's `np.ndarray`.

This should be solved by using an `isinstance()` check instead, i.e., `assert isinstance(res, type(convert_input))`.